src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="index.html" method="get">
+      <div class="form-big" data-qa="big">
+        <label class="
+        form__loop
+        form__loop--size_big
+        form-big__big-loop"
+        for="big-loop"
+        >
+        </label>
```

Comment:
check it in other places
```suggestion
        <label class="
          form__loop
          form__loop--size_big
          form-big__big-loop"
          for="big-loop"
        >
        </label>
```